README.md

```diff
@@ -89,6 +89,7 @@ Name | Description
 [dotnet-bundle](https://github.com/madskristensen/BundlerMinifier) | BundlerMinifier.Core let's you configure bundling and minification of JS, CSS and HTML files. [NuGet](https://www.nuget.org/packages/BundlerMinifier)
 [dotnet-clean](https://github.com/tsolarin/dotnet-clean) | DotNet.Cleaner.Tools is a tool for cleaning projects [NuGet](https://nuget.org/packages/DotNet.Cleaner.Tools)
 [dotnet-commands](https://github.com/Lambda3/dotnet-commands) | A tool that allows you to use any executable as a .NET CLI Command, with special treatment for .NET Core apps. Follow [these instructions](https://github.com/Lambda3/dotnet-commands) to install.
+[dotnet-config2json](https://github.com/andrewlock/dotnet-config2json) | A simple tool to convert a web.config file to an appsettings.json file. [NuGet](https://www.nuget.org/packages/dotnet-config2json/)
```

Comment:
Can you put this in the first table in the README? This one is for project-tools (e.g. DotNetCliToolReference), but I think your tool is a global tool (`dotnet tool install -g`)